src/pg/test/expected/40_observatory_utility_test.out

```diff
@@ -0,0 +1,79 @@
+SELECT set_config(
```

Comment:
Ok, it's up there now! 
https://github.com/CartoDB/observatory-extension/commit/b3b4c8e4c303d9beea50f9c51cb0228260e38a65